src/components/scenes/Staking/StakeOverviewScene.js

```diff
@@ -36,12 +37,15 @@ export const StakeOverviewScene = (props: Props) => {
   const theme = useTheme()
   const styles = getStyles(theme)
 
-  const currencyWallet = useSelector((state: RootState) => state.core.account.currencyWallets[walletId])
+  const account = useSelector(state => state.core.account)
```

Comment:
Since you only need the seed, which is always on the wallet and never change, you just use the selector as before without watching on the account.